### DISCUSSION TITLE
windows安装版安装位置

### DISCUSSION THREAD

**Discussion by Vatario** [qier222/YesPlayMusic#1398] [Q&A] [answered]
windows安装版不可以自定义安装位置吗……?缓存全缓存在c盘有点顶不住

**Comment by memorydream** [comment, marked as answer]
安装目录无法更改，但是可以下载单文件版并用压缩软件打开，解压里面ypm相关的文件到任意目录
缓存目录受限于实现方式无法更改，可以在ypm的设置中调整可用的缓存大小或关闭缓存
